common/djangoapps/django_sudo/utils.py

```diff
@@ -0,0 +1,32 @@
+"""
+sudo.utils
+"""
+import django.contrib.sessions.middleware
+import sudo.middleware
+
+
+def region_name(region):
```

Comment:
Should this just be an internal function of the `django_sudo` 3rd party library to sanitize region names? The fact that it's being stored as a cookie is part of the `django_sudo` code, right?